db/models.js

```diff
@@ -77,9 +79,56 @@ const deleteById = (id, callback) => {
     });
 };
 
+// use for now
+const createFakeAuthors = () => ({
+  name: faker.name.findName(),
+  followers: faker.random.number(),
+  biography: faker.lorem.paragraph(),
+  author_image: faker.image.people(),
+  createdAt: moment(faker.date.past()).format('YYYY-MM-DD HH-mm-ss'),
+  updatedAt: moment(faker.date.past()).format('YYYY-MM-DD HH-mm-ss'),
+});
+
+const createFakeBooks = () => ({
+  title: faker.commerce.productName(),
+  year: faker.random.number({ min: 1900, max: 2019 }),
+  total_ratings: faker.random.number(),
+  average_rating: faker.random.number({ min: 1, max: 5 }),
+  description: faker.lorem.paragraph(),
+  cover_image: faker.image.city(),
+  author_id: faker.random.number({ min: 1, max: 100 }),
+  createdAt: moment(faker.date.past()).format('YYYY-MM-DD HH-mm-ss'),
+  updatedAt: moment(faker.date.past()).format('YYYY-MM-DD HH-mm-ss'),
+});
+
```

Comment:
In my opinion, createFakeAuthors & createFakeBooks methods do not belong in this module. The module should receive the data and insert into the database